<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>GamePlayingState</class>
    <widget class="QWidget" name="GamePlayingState">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1222</width>
                <height>819</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="styleSheet">
            <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,10">
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_4">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_3">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QPushButton" name="abdicateButton">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>150</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="text">
                                        <string>Please wait...</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_5">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_3">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_2">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <widget class="QWidget" name="widget_3" native="true">
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <layout class="QHBoxLayout" name="horizontalLayout_15">
                                        <property name="topMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item>
                                            <layout class="QHBoxLayout" name="horizontalLayout_3"/>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item>
                                <widget class="QWidget" name="widget_5" native="true">
                                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                                        <property name="leftMargin">
                                            <number>9</number>
                                        </property>
                                        <property name="topMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item>
                                            <layout class="QHBoxLayout" name="horizontalLayout_2">
                                                <property name="spacing">
                                                    <number>20</number>
                                                </property>
                                            </layout>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item>
                                <widget class="QWidget" name="widget" native="true">
                                    <property name="styleSheet">
                                        <string notr="true">background-color: rgba(255, 255, 127,255);
                                            background-color: rgba(170, 170, 127,0);
                                        </string>
                                    </property>
                                    <layout class="QHBoxLayout" name="horizontalLayout_16">
                                        <property name="topMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item>
                                            <layout class="QHBoxLayout" name="horizontalLayout_6">
                                                <property name="spacing">
                                                    <number>20</number>
                                                </property>
                                            </layout>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item>
                                <widget class="QWidget" name="widget_4" native="true">
                                    <layout class="QHBoxLayout" name="horizontalLayout_17">
                                        <property name="topMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item>
                                            <layout class="QHBoxLayout" name="horizontalLayout_4">
                                                <property name="spacing">
                                                    <number>20</number>
                                                </property>
                                            </layout>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item>
                                <widget class="QWidget" name="widget_6" native="true">
                                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                                        <property name="topMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item>
                                            <layout class="QHBoxLayout" name="horizontalLayout_7">
                                                <property name="spacing">
                                                    <number>20</number>
                                                </property>
                                            </layout>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item>
                                <widget class="QWidget" name="widget_7" native="true">
                                    <layout class="QHBoxLayout" name="horizontalLayout_10">
                                        <property name="topMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item>
                                            <layout class="QHBoxLayout" name="horizontalLayout_9">
                                                <property name="spacing">
                                                    <number>20</number>
                                                </property>
                                            </layout>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item>
                                <widget class="QWidget" name="widget_8" native="true">
                                    <layout class="QHBoxLayout" name="horizontalLayout_12">
                                        <property name="topMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item>
                                            <layout class="QHBoxLayout" name="horizontalLayout_11">
                                                <property name="spacing">
                                                    <number>20</number>
                                                </property>
                                            </layout>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item>
                                <widget class="QWidget" name="widget_2" native="true">
                                    <layout class="QHBoxLayout" name="horizontalLayout_14">
                                        <property name="topMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item>
                                            <layout class="QHBoxLayout" name="horizontalLayout_13"/>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
